udata/core/dataset/croquemort.py

```diff
@@ -64,8 +64,8 @@ def check_url_from_cache(url, group=None):
         response = requests.get(retrieve_url,
                                 params={'url': url, 'group': group},
                                 timeout=TIMEOUT)
-    except requests.ConnectionError:
-        return {'error': 'URL checker not found'}, {}
+    except requests.RequestException:
```

Comment:
A 404 is returned when the URL is not found on the server, not when the server is not up and running. I think that we should differentiate `ConnectionError` and `ReadTimeout` (and maybe fallback on `RequestException`?). The first one happens when there is a server error, the second when there is a network error. Having that information can help us debug the issue more fast.